ko/documentation/quickstart/2/index.md

```diff
@@ -65,7 +65,7 @@ Hello Matz!
 
 ## 문자열의 중간에 자리를 잡아두기
 
-`#\{name}` 부분은 무엇이냐구요? Ruby에서 문자열 중간에 무엇을 삽입하는 방법입니다.
```

Comment:
https://github.com/search?q=repo%3Aruby%2Fwww.ruby-lang.org%20%23%5C%7B&type=code
Code quotes seem to have been added later. There are traces of them in other languages ​​as well.